library_generation/DEVELOPMENT.md

```diff
@@ -144,16 +152,21 @@ repo to this folder).
 
 To run the docker container on the google-cloud-java repo, you must run:
 ```bash
-docker run -u "$(id -u)":"$(id -g)"  -v/path/to/google-cloud-java:/workspace $(cat image-id)
+docker run \
+  -u "$(id -u)":"$(id -g)"  \
+  -v /path/to/google-cloud-java:/workspace \
+  $(cat image-id)
 ```
 
  * `-u "$(id -u)":"$(id -g)"` makes docker run the container impersonating
    yourself. This avoids folder ownership changes since it runs as root by
    default.
- * `-v/path/to/google-cloud-java:/workspace` maps the host machine's
-   google-cloud-java folder to the /workspace folder. The image is configured to
-   perform changes in this directory
- * `$(cat image-id)` obtains the image ID created in the build step
+ * `-v /path/to/google-cloud-java:/workspace` maps the host machine's
+   google-cloud-java folder to the /workspace folder. 
+   The image is configured to perform changes in this directory.
+   Note that this setup assumes the api definition resides in host machine's
```

Comment:
Can we change this example to use the `api_definition_path` parameter?

Reply:
Done.